docs/DeveloperGuide.adoc

```diff
@@ -721,6 +721,30 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
+|`* * *` |busy user |search using accumulative filters |narrow down my search in the shortest time possible
+
+|`* * *` |careless user that cant remember the order of entry |add a contact without following the particular order of entry |not remember the order of entry
+
+|`* * *` |forgetful user that cant remember all tags |view a list of existing tags |recall the tags associated with a person
```

Comment:
"can't"
I think the benefit is that we can recall and reuse tags we have used? So we won't have duplicate tags that are not exactly the same, but mean the same thing, like "cs2103" and "CS2103"